diff --git a/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h b/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
index ccc6510..b78f21c 100644
--- a/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
+++ b/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
@@ -8,6 +8,7 @@
 #pragma once
 
 #include <array>
+#include <string>
 #include <unordered_map>
 
 #include <glog/logging.h>
@@ -68,7 +69,7 @@ inline folly::dynamic toDynamic(const YGValue &dimension)
     case YGUnitPoint:
       return dimension.value;
     case YGUnitPercent:
-      return std::format("{}%", dimension.value);
+      return std::to_string(dimension.value) + "%";
   }
 
   return nullptr;
